src/h5web/toolbar/MeasuredControl.tsx

```diff
@@ -4,23 +4,29 @@ import styles from './Toolbar.module.css';
 
 interface Props {
   children: ReactElement;
+  knownWidth: number | undefined;
   onMeasure: (width: number) => void;
 }
 
 function MeasuredControl(props: Props) {
-  const { children: child, onMeasure } = props;
+  const { children: child, knownWidth, onMeasure } = props;
 
   return (
     <Measure
       onResize={({ entry }: { entry?: DOMRect }) => {
-        if (entry) {
+        if (entry && entry.width > (knownWidth || 0)) {
```

Comment:
This is the line that fixes the jumping in/out issue. Much more elegant than if I had to do it in `Toolbar`.